src/get-started/java-binding.md

```diff
@@ -519,10 +539,16 @@ The TestKit can be also instantiated using a builder, if different
 configuration is needed:
 
 ```java
-try (TestKit testKit = TestKit.builder()
-    .withServices(MyServiceModule.class, MyServiceModule2.class)
-    .withValidators(2)
-    .build()) {
+// This TestKit will be instantiated with two service instances of different
+// services `ARTIFACT_ID` and `ARTIFACT_ID_2`
```

Comment:
Shall we put somewhere a note what those constants in the following examples are and where to get them (either hardcode or pass through system properties or configuration files)?